errors/error-handlers.js

```diff
@@ -0,0 +1,23 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status) {
+      res.status(err.status).send({ msg: err.msg });
+    } else {
+      next(err);
+    }
+  };
+  
+  exports.handlePsqlErrors = (err, req, res, next) => {
+    if (err.code === "22P02" || err.code === "23502") {
+      res.status(400).send({ msg: "bad request" });
+    } else {
+      next(err);
+    }
+  };
+  exports.handleServerErrors = (err, req, res, next) => {
+    console.log(err);
+    res.status(500).send({ msg: "internal server error" });
+  };
```

Comment:
please try follow TDD and only add the code needed to pass the current failing test.